doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -402,6 +402,20 @@ inputs.
 | `replica_id` | [`text`]    | The ID of a cluster replica. |
 | `hydrated`   | [`boolean`] | Whether the object is hydrated on the replica. |
 
+## `mz_license_keys`
+
+The `mz_license_keys` table describes the license keys which are currently in
+use.
+
+<!-- RELATION_SPEC mz_internal.mz_license_keys -->
+| Field            | Type                         | Meaning  |
+| -----------      | -----------                  | -------- |
+| `id`             | [`text`]                     | The identifier of this particular license key. |
+| `organization`   | [`text`]                     | The name of the organization that this license key was issued to. |
+| `environment_id` | [`text`]                     | The environment ID that this license key was issued for. |
```

Comment:
Also, just for my edification  ... what does the environment_id map to? (heh heh ... I'm sure it's in some design doc somewhere)

Reply:
environment_id is the use case id which corresponds to the `--environment-id` flag passed to environmentd